### PR TITLE
Add a TAG Makefile variable for the kube2sky image

### PR DESCRIPTION
**Pull request by satnam6502**

### CODE REVIEW

File: cluster/addons/dns/kube2sky/Makefile [hunk 1]
```diff
@@ -1,13 +1,21 @@
-all: kube2sky
+# Makefile for the Docker image gcr.io/google_containers/kube2sky
+# MAINTAINER: thockin@google.com
+# If you update this image please bump the tag value before pushing.
+
+.PHONY: all kube2sky container push clean
+
+TAG = 1.4
+
+all: kube2sky container push
```
Comment (thockin) [inline cluster/addons/dns/kube2sky/Makefile]: I don't like 'push' being part of all.  I think that should be a manual step.

File: cluster/addons/dns/kube2sky/Makefile [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
 
 kube2sky: kube2sky.go
 	CGO_ENABLED=0 godep go build -a -installsuffix cgo --ldflags '-w' ./kube2sky.go
 
 container: kube2sky
-	docker build -t gcr.io/google_containers/kube2sky .
+	docker build -t gcr.io/google_containers/kube2sky:$(TAG) .
```
Comment (thockin) [inline cluster/addons/dns/kube2sky/Makefile]: maybe we should parameterize the prefix, too?  When I was testing I would push to thockin/kube2sky and run that.

File: cluster/addons/dns/kube2sky/Makefile (hunk 1 above)
Comment (satnam6502) [inline cluster/addons/dns/kube2sky/Makefile]: Fixed.

File: cluster/addons/dns/kube2sky/Makefile (hunk 2 above)
Comment (satnam6502) [inline cluster/addons/dns/kube2sky/Makefile]: Done.

**Comment by satnam6502** [comment]
PTAL.

**Comment by satnam6502** [comment]
@cjcullen do you know how the kube2sky image go its 1.4 tag before?

**Comment by cjcullen** [comment]
@satnam6502 I manually added it :)

**Pull request merged**